content/react/concepts/react-native/react-native-components/react-native-components.md

```diff
@@ -0,0 +1,160 @@
+---
+Title: 'React Native Components'
```

Comment:
@Dusch4593 I'm wondering if maybe we can use the term "Component" instead of views. For React Native it looks like View is one of the many Components that can be utilized

Reply:
Maybe we can use "Native Components" instead? View is one of the native components so this may cause some confusion. Also, the images added support this term usage and provides the iOS/Android parallel.

Reply:
Totally cool with that 😄